index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,700;1,300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">     
+    <title>Personal-Portfolio</title>
+  </head>
+
+  <body>
```

Comment:
Please do not add blank lines or spaces without a reason. Add blank lines to separate large or logical code blocks. 💪